t/Geometry.t

```diff
@@ -7,17 +7,72 @@ use DDG::Test::Spice;
 
 ddg_spice_test(
     [qw( DDG::Spice::Geometry )],
-    'geometry square' => test_spice(
+    #tests for queries that display all formulas of a given shape
+    'geometry sphere' => test_spice(
         '/js/spice/geometry/',
         call_type => 'self',
         caller => 'DDG::Spice::Geometry'
     ),
-    'area rectangle formula' => test_spice(
+    'formulas square' => test_spice(
```

Comment:
@GuiltyDolphin I would do that, but I am not sure how to write test files for tests that are not supposed to trigger the spice. Can you please point me to some page where I can read about it?
Thanks!